src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/SearchPage/SearchAdvanced.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component, PropTypes } from "react";
+import Collapse from "dnn-collapsible";
+import { SearchIcon } from "dnn-svg-icons";
+import Localization from "../../localization";
+import SearchAdvancedDetails from "./SearchAdvancedDetails";
+import "./styles.less";
+
+
+class SearchAdvanced extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            collapsed:false
+        };
+    }
+
+    toggle() {
+        this.setState({
+            collapsed:!this.state.collapsed
+        });
+    }
+
+    onClear() {
+        this.setState({
+            collapsed : false
+        });
+    }
+
+    /*eslint-disable react/no-danger*/
+    render() {
+        return (
+            <div className={`advancedCollapsibleComponent ${this.state.collapsed?"open":""}`}>
```

Comment:
This is what we had before, when it was a pop-up. Because when on open/close, or it would come back to pop-up or it would slide all header to accommodate the Advanced Filter.